### ISSUE TITLE
Minecraft textures

### ISSUE THREAD

**Issue by basxto** [ZetaGlest/zombie_invasion#3]
I only checked a few sprites out (cow), but they look like they are directly based on the original minecraft textures. https://minecraft.net/en-us/article/try-new-minecraft-java-textures
Which would be copyright infringement.

If that’s the case, please take a look at this project: http://repo.or.cz/MineClone/MineClone2.git

**Comment by andy5995** [comment]
Thank you so much for the FYI @basxto . I think we'll just delete this repo and the maintainer can host it on his own account instead, until it's all sorted out.

**Comment by basxto** [comment]
I’ve started replacing unit textures yesterday, but it’s likely better to start ne github project (without this issue) and modify the original commit.
Problematic is that there were no license included in the original release.
Only unit textures I haven't managed to replace so far:
* chest
* diamond_steve
* iron_golem
* iron_steve
* sheep
* skeleton (just the bow)
* steve
* spider_jockey (depends on skeleton)

also missing a replacement for:
* loading_screen.jpg

**Comment by andy5995** [comment]
>I’ve started replacing unit textures yesterday, but it’s likely better to start ne github project (without this issue) and modify the original commit.

Agreed. @linux-player9 is the maintainer of this mod, so I'll just give him a quick ping xD
